src/main/java/at/ac/tuwien/damap/rest/ConfigResource.java

```diff
@@ -44,7 +50,12 @@ public ConfigDO config() {
         configDO.setAuthUser(authUser);
         configDO.setEnv(env);
         configDO.setPersonSearchServiceConfigs(personServiceConfigurations.getConfigs());
+        configDO.setFitsServiceAvailable(getFitsServiceAvailability());
 
         return configDO;
     }
+
+    private Boolean getFitsServiceAvailability() {
+        return fitsUrl.isPresent() && !fitsUrl.toString().isEmpty();
```

Comment:
URL performs some checks on the provided string (e.g. protocol, address) and will throw, if it is not a valid URL. Therefore, the second part of the condition can be omitted. What do you think?